### ISSUE TITLE
bug in parser/models/ConceptModel.py

### ISSUE THREAD

**Issue by zjxs1997** [ChunchuanLv/AMR_AS_GRAPH_PREDICTION#8]
Hello, thank you for providing your codes. But when I ran the test command using your pre-trained model `python src/generate.py -train_from gpus_0valid_best.pt`, something went wrong.
I found that in line 135, ConceptModel.py, the code calls the forward function of encoder (which is an instance of Class SentenceEncoder), with only PackedInput as input parameter, and causes the second parameter hidden setted as default `None`, and this finally causes the problem in line 65 `self.rnn(emb, hidden)`.
Could you please help me out with this problem? Thank you so much.

**Comment by ChunchuanLv** [comment]
The pretrained model might be bugged for other reason, but hidden = none
will only force the lstm to initialize with zeros  I suppose.

Could you train a new model ?

**Comment by zjxs1997** [comment]
I have successfully trained a model using pytorch1.0 and cuda9, but during training it is very easy to collapse (seemed like NaN issue, I am not sure).
By the way, the size of `emb` in line 65 is also not appropriate, lstm takes 3-dim tensor as input, but the data of emb is 2-dim.


> The pretrained model might be bugged for other reason, but hidden = none will only force the lstm to initialize with zeros I suppose. Could you train a new model ?
